.github/workflows/android.yml

```diff
@@ -70,10 +70,17 @@ jobs:
         run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
 
       - name: Save git version so the app can show it to users
-        env:
-          # we need this hack because github default env vars don't include the tag itself
-          GIT_TAG: ${{ steps.vars.outputs.tag }}
```

Comment:
why did you remove these lines? Don't we need the GIT_TAG anymore? It is still used in the run command below. You also did not remove it in the ios.yml file.

Reply:
I don't know what happened here, but it should not have gone. Thanks for picking it up!